zipline/assets/assets.py

```diff
@@ -703,6 +704,10 @@ def lookup_symbol(self, symbol, as_of_date, fuzzy=False):
             there are multiple candidates for the given ``symbol`` on the
             ``as_of_date``.
         """
+        if symbol is None:
+            raise TypeError("Cannot lookup symbol of type NoneType for "
+                            "as of date %s" % as_of_date)
```

Comment:
Why do we include `as_of_date` here?  What about `fuzzy`?  Or could we just say "Cannot lookup None symbol"?